### PR TITLE
docs(preset): Add Jetpack Preset 

### PR DESCRIPTION
**Pull request by Indyandie**
# Jetpack Preset

#### Description
This is a pseudo minimalist preset inspired by the [geometry](https://github.com/geometry-zsh/geometry) and [spaceship](https://github.com/spaceship-prompt/spaceship-prompt) prompts.

#### Motivation and Context
Preset

#### Screenshots
jetpack

#### How Has This Been Tested?
- basic usage
- git
- [x] I have tested using **MacOS**
- [x] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly. (I hope so. 😓 )
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: docs/presets/jetpack.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- [Jetbrains Mono](https://www.jetbrains.com/lp/mono/) installed and enabled in the terminal.
+
+### Manual Configuration
+
+[Click to download TOML](/presets/toml/jetpack.toml)
+
+### Configuration
+
+```sh
+starship preset jetpack-o ~/.config/starship.toml
```
Comment (mickimnet) [inline docs/presets/jetpack.md:22]: There is a typo in this line: you forgot a space between `… jetpack` and `-o`. It should be `starship preset jetpack -o ~/.config/starship.toml`.

File: docs/presets/jetpack.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+```
+
```
Comment (mickimnet) [inline docs/presets/jetpack.md:24]: All other presets offer a line to download the _TOML_ if one does prefer. I would recommend adding it—see [Nerd Font Symbols Preset](https://starship.rs/presets/nerd-font.html).

**Review by mickimnet — CHANGES_REQUESTED** [review]

File: docs/presets/jetpack.md (hunk 2 above)
Comment (Indyandie) [reply docs/presets/jetpack.md:25]: @mickimnet Did I use the incorrect format for downloads in [line 19](https://github.com/Indyandie/starship/blob/bc255473b4371bbbf3e23b5fbd689403a1c9f175/docs/presets/jetpack.md?plain=1#L18)?

**Comment by Indyandie** [comment]
@mickimnet thanks so much for the review. 🙇 The only thing that's outstanding is the [download link](https://github.com/starship/starship/pull/5261/files#r1271444804).

**Comment by Indyandie** [comment]
NVM I gotta fix these conflicts first.

File: docs/presets/jetpack.md (hunk 2 above)
Comment (mickimnet) [reply docs/presets/jetpack.md:25]: I'll check later today or tomorrow.

File: docs/presets/jetpack.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+### Manual Configuration
+
+[Click to download TOML](/presets/toml/jetpack.toml)
+
+### Configuration
+
+```sh
+starship preset jetpack -o ~/.config/starship.toml
+```
+
```
Comment (mickimnet) [inline docs/presets/jetpack.md:25]: To be in line with the documentation of the other presets, I suggest:

```suggestion
### Configuration

```sh
starship preset jetpack -o ~/.config/starship.toml
```

```suggestion
[Click to download TOML](/presets/toml/jetpack.toml)
```

If the suggestions don't render correctly: put the TOML link just below the command line section.

**Review by mickimnet — CHANGES_REQUESTED** [review]
Additional to the code suggestion, please remove all `src/` files from your commit.

**Comment by mickimnet** [comment]
There is still `src/modules/git_status.rs` in the pull request. Maybe that is the reason, why the checks are failing.

**Comment by Indyandie** [comment]
testing checks in #5347

**Comment by Indyandie** [comment]
@mickimnet looks like the checks are passing now... 😮‍💨

**Review by mickimnet — APPROVED** [review]
LGTM.

File: docs/.vuepress/public/presets/toml/jetpack.toml [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+$time\
+"""
+
+add_newline = true
+
+[character]
+format = "$symbol "
+success_symbol = "[◉](bold italic bright-yellow)"
+error_symbol = "[⊘](italic purple)"
+# vicmd_symbol = "[◈](italic white)"
```
Comment (davidkna) [inline docs/.vuepress/public/presets/toml/jetpack.toml:96]: Why are there commented options in the preset?

File: docs/.vuepress/public/presets/toml/jetpack.toml [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+style_root = "purple bold italic"
+format = "[$user]($style) ▻ "
+disabled = false
+
+[directory]
+home_symbol = "⌂"
+truncation_length = 2
+truncation_symbol = "▦ "
+read_only = " ■"
+use_os_path_sep = true
```
Comment (davidkna) [inline docs/.vuepress/public/presets/toml/jetpack.toml:119]: That's the default and not a strong requirement for the theme:
```suggestion
```

File: docs/.vuepress/public/presets/toml/jetpack.toml [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+style = "bright-yellow italic"
+
+[swift]
+format = " sw [${symbol}${version}]($style)"
+symbol = "◁ "
+style = "italic bright-red"
+version_format = "${major}.${minor}"
+
+[aws]
+disabled = true
```
Comment (davidkna) [inline docs/.vuepress/public/presets/toml/jetpack.toml:228]: If you're providing theming for the module, I don't see why it should be disabled if it's usually enabled by default.

**Review by davidkna — CHANGES_REQUESTED** [review]

File: docs/.vuepress/public/presets/toml/jetpack.toml (hunk 4 above)
Comment (Indyandie) [reply docs/.vuepress/public/presets/toml/jetpack.toml:96]: For some reasons checks failed with those options, and I forgot to remove it. I'll remove it.

**Review by davidkna — APPROVED** [review]

**Comment by chipbuster** [comment]
Thank you to @davidkna and @mickimnet for reviews, and to @Indyandie for writing this!

**Pull request merged**